docs/user/index.rst

```diff
@@ -108,11 +108,14 @@ to help you create fantastic documentation for your project.
    :caption: Reference
    :glob:
 
+   /reference/features
+   /glossary
    /api/index
    /server-side-search/api
-   /reference/features
    /server-side-search/syntax
-   /reference/policies
+   changelog
+   Developer Documentation <https://dev.readthedocs.io>
```

Comment:
This is a good idea... I really wish we had an easy way to put an external link icon on this... do we? https://fontawesome.com/v5/icons/external-link-alt?s=solid&f=classic

Reply:
We can use an emoji for this, for instance 🖇️ or 🔗️ -- I think the latter is the most conventional one. https://emojipedia.org/link/

Reply:
Yea, but really the tool should probably enforce this?

Reply:
As in the theme should add an icon or Sphinx should add a CSS class that could be styled?

Reply:
Yep. But baby steps :)